src/components/DropList/DropList.css.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react'
+import styled from 'styled-components'
+import { getColor } from '../../styles/utilities/color'
+import { FONT_FAMILY as AKTIV_FONT_FAMILY } from '../HSDS/GlobalStyle'
+import Icon from '../Icon'
+
+export const DropListWrapperUI = styled('div')`
+  box-sizing: border-box;
+  width: ${({ variant }) => (variant === 'combobox' ? '220px' : '200px')};
+  padding: 0;
+  background-color: white;
+  border: 1px solid ${getColor('grey.600')};
+  border-radius: 4px;
+  box-shadow: 0px 1px 7px rgba(0, 0, 0, 0.08);
+  font-family: ${AKTIV_FONT_FAMILY};
```

Comment:
instead of forcing Aktiv here, maybe it should it be `var(--HSDSGlobalFontFamily)` . Just in case the font is overwritten on the global stylesheet. 

same with the font-size : `var(--HSDSGlobalFontSize)`.

Reply:
Always forget... thanks!